### PR TITLE
[clang][NFC] Fix arm_acle.h title headers

### PR DESCRIPTION
**Pull request by vhscampos**
Fix some title headers to align them with the actual ACLE document.

### CODE REVIEW

**Review by jwestwood921 — APPROVED** [review]
LGTM, but wait for @DavidSpickett to review before merging.

**Review by DavidSpickett — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
